test/e2e/rm_test.go

```diff
@@ -139,9 +139,23 @@ var _ = Describe("Podman rm", func() {
 		Expect(podmanTest.NumberOfContainers()).To(Equal(1))
 
 	})
+
 	It("podman rm bogus container", func() {
 		session := podmanTest.Podman([]string{"rm", "bogus"})
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(1))
 	})
+	It("podman rm bogus container and a running container", func() {
+		session = podmanTest.RunTopContainer("test1")
```

Comment:
needs to be `session := `

Reply:
Oops! Sorry, I keep making this mistake. Fixed.